gtsfm/utils/io.py

```diff
@@ -290,6 +290,16 @@ def read_cameras_txt(
                 float(k1),
                 float(k2),
             )
+        elif model == "PINHOLE":
```

Comment:
could be nice to include an inline comment here that
# The 'PINHOLE' model is defined as a 4-parameter camera model: fx, fy, cx, cy
# It is defined in https://github.com/colmap/colmap/blob/main/src/colmap/sensor/models.h

Reply:
Gentle bump on this